.github/workflows/code-formatting.yml

```diff
@@ -7,6 +7,21 @@ on:  # Trigger the workflow on push or pull request, but only for the master bra
     branches: [master]
 
 jobs:
+  isort:
```

Comment:
Could you add `isort` to pre-commit too. Here is an example: https://github.com/PyCQA/isort/blob/develop/.pre-commit-config.yaml

Reply:
I already did actually :wink: But I asked @Borda about it, and he wanted to keep the pre-commit hook for a different PR. When you're ready for it, you can just ping me though, because I already have it locally!

Reply:
yes as with accepting the CI check we need to format the codebase accordingly...